packages/contract/src/interfaces/IDaimoPayBridger.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.12;
+
+import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
+/// @notice Bridges assets automatically. Specifically, it lets any market maker
+/// initiate a bridge transaction to another chain.
+interface IDaimoPayBridger {
+    /// @notice Emitted when a bridge transaction is initiated
+    event BridgeInitiated(
+        uint256 indexed toChainId,
+        address indexed toAddress,
```

Comment:
maybe not indexed unless there's a reason for them to be -> costs a bit more gas